### DISCUSSION TITLE
New widget Idea: CTkRadarChart

### DISCUSSION THREAD

**Discussion by Akascape** [thisal-d/ctkchart#7] [Ideas]
@Thisal-D I have a new widget idea for you, can you make some type of radar chart for tkinter?

**It looks something like this:**
![image](https://th.bing.com/th/id/OIP.uFYrySdJRuhkHFADVn7WuAAAAA?rs=1&pid=ImgDetMain)
![image](https://github.com/Thisal-D/ctkchart/assets/89206401/37f54206-1511-4c3a-9bce-7c983b3bc987)

**Comment by thisal-d** [comment]
> @Thisal-D I have a new widget idea for you, can you make some type of radar chart for tkinter?
> 
> **It looks something like this:**
> ![image](https://th.bing.com/th/id/OIP.uFYrySdJRuhkHFADVn7WuAAAAA?rs=1&pid=ImgDetMain)
> ![image](https://github.com/Thisal-D/ctkchart/assets/89206401/37f54206-1511-4c3a-9bce-7c983b3bc987)
> 

It seems possible 🙂 , I will try, but not sure.

Btw I added a new library to your tkinter tool. Check it out,

**Reply by Akascape** [reply]
@Thisal-D It's not so difficult to draw this chart imo, it will be a great addition. There is no need to color the area after plotting, because it will overlap the bg lines.  You can try to make something like the first image I shown.

> Btw I added a new library to your tkinter tool. Check it out,

I have added the library, thanks

**Reply by thisal-d** [reply]
Sure bro, I'll do it,

**Comment by Akascape** [comment]
@Thisal-D Any progress on this? 😅

**Reply by thisal-d** [reply]
I didn't even start bro🙂🫠, sorry

**Reply by Akascape** [reply]
@Thisal-D Oh, I thought you have made the widget as you closed this discussion. No worries, take your time.

**Reply by thisal-d** [reply]
> @Thisal-D Oh, I thought you have made the widget as you closed this discussion. No worries, take your time. 

Are you in a hurry? Im sorry. I have other work to do bro.😑

**Reply by Akascape** [reply]
@Thisal-D Have you started this project?

**Comment by thisal-d** [comment]
@Akascape nah bro i didn't

**Comment by Akascape** [comment]
@Thisal-D I made the widget with a little help from bing copilot and it looks like this:
https://github.com/Akascape/CTkRadarChart

![screenshot](https://github.com/Akascape/CTkRadarChart/assets/89206401/0d3ecda5-f73d-4d27-b7d7-817cf42905ec)

**Reply by thisal-d** [reply]
@Akascape it's nice 🙂
